modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/object-definitions/koroneiki-account.object-entries.json

```diff
@@ -47,7 +47,8 @@
 		"partner": false,
```

Comment:
Hey, Lucas! I can't see this data mocks on the website. Can you check this?

Reply:
Thanks @joanagrego. I clarifed on _"Description"_
You need drop the database before up tomcat.